### PR TITLE
Update 2012-04-04-README.md

### PR DESCRIPTION
**Pull request by Mamizou**

### CODE REVIEW

File: nights/2012-04-04-README.md [hunk 1]
```diff
@@ -21,14 +21,8 @@ Check these off if they are true:
 - [ ] No images are missing filter information (except BIAS and DARK, which need no filter).
```
Comment (vd0174yl) [inline nights/2012-04-04-README.md]: Please fill in these checkboxes [ ] by putting an x in them.

**Comment by Mamizou** [comment]
@mwcraig Please add this night to the server.

**Comment by Mamizou** [comment]
closes #294

**Comment by Mamizou** [comment]
Closes #294

**Pull request closed without merging**